src/Writer/CSV/Writer.php

```diff
@@ -28,6 +28,11 @@ public function getOptions(): Options
     {
         return $this->options;
     }
+    
+    public function setCreator(string $creator): void
+    {
+        // noop
```

Comment:
May it would be better to raise an Exception message like `Method unsopported for CSV documents`